updatecli/updatecli.d/update-plugins.yaml

```diff
@@ -59,7 +59,19 @@ sources:
         kind: regex
         pattern: 'jenkins-2.332.\d+$'
     transformers:
-      - trimPrefix: "jenkins-"
+      - trimprefix: "jenkins-"
+  jenkins2.346.x:
+    name: Get Last Jenkins Stable Version
+    kind: githubrelease
+    spec:
+      owner: "jenkinsci"
+      repository: "jenkins"
+      token: "{{ requiredEnv .github.token }}"
+      versionfilter:
+        kind: regex
+        pattern: 'jenkins-2.346.1$' # fix to x.x.1 until the next stable minor release
```

Comment:
relatively easy to bump the version pattern to `.\d+$` and add the next stable minor release and add the new stable release pinned to `.1$`